n2y/plugins/hiddenjinjatoggles.py

```diff
@@ -12,7 +12,7 @@
 class HiddenJinjaToggleBlock(ToggleBlock):
```

Comment:
Can you rename this to "ExpandBlueToggleBlock". Also can you rename the file to "expandbluetoggles.py"